src/script/page/state.ts

```diff
@@ -52,7 +80,152 @@ type AppMainState = {
   };
 };
 
+const legalHoldModalDefaultState = {
+  conversation: null,
```

Comment:
i know you left a comment on wire, but maybe its good to make a comment once here as well, for anyone who misses the other one. you said you will fix these any type tests failing.